source/extensions/filters/http/aws_lambda/aws_lambda_filter.cc

```diff
@@ -19,6 +19,10 @@
 
 #include "source/extensions/filters/http/solo_well_known_names.h"
 
+#include "nlohmann/json.hpp"
+// For convenience
+using json = nlohmann::json;
```

Comment:
How is this used / why was it not needed when we only supported response transformations?

Reply:
This is not necessary here -- I was using it when I had a temporary transformer defined here.

Reply:
Removed this here: cb39433a0958a4bec1cae76b686fe1fbbe5ec390